source/Classroom/Basics/Billing/how_to_read_your_invoice.md

```diff
@@ -1,86 +1,69 @@
 ---
 seo:
-  title: How to read your invoice
-  description: How to read your invoice
+  title: Reading your invoice
+  description: Reading your invoice
   keywords: invoice, bill, receipt, refund, line, item, marketing, campaigns, charge, contacts, usage, overage
-title: How to read your invoice
+title: Reading your invoice
 weight: 0
 layout: page
 zendesk_id: 
 navigation:
   show: true
 ---
-When you receive your invoice on the 1st of each month, you might have questions. This article aims to clarify what your invoices mean, and to help you learn more about what you are paying for.
 
-![]({{root_url}}/images/invoice_legend.png)
+SendGrid invoices can be confusing, particularly if you've made changes to your account during the previous month.
 
- Please refer to the legend below for more information on each of these items. 
+Here's more information about reading your invoice: 
 
- 
+![]({{root_url}}/images/invoice_legend.png)
 
-# 1. Physical Address
+{% anchor h2 %}
+1. Physical Address
+{% endanchor %}
 
 **The physical address of your business.** This address is the same address you entered when you first signed up. If you would like to update the address as it appears on your invoices, you can click on the "Change Invoice Address" button on the [Plan & Billing Details](https://app.sendgrid.com/settings/billing) page. If you need to change the address on an existing invoice, please send an email to [billing@sendgrid.com](mailto:billing@sendgrid.com).
```

Comment:
If you would like to update the address as it appears on your invoices, click **Change Invoice Address** on the [Plan & Billing Details] page.